edx_proctoring/static/proctoring/templates/add-new-allowance.underscore

```diff
@@ -0,0 +1,56 @@
+<div class='modal-header'><%- gettext("Add a new Allowance") %></div>
```

Comment:
Should we follow the pattern for other "Add" buttons that are in the LMS  Instructor Dashboard? An example is the "Add Cohort" button under "Cohorts" that appears when you select "Enable Cohorts". Typically there is a plus sign and then text. In this case it would be "+ Add Allowance".
@chrisndodge @talbs?